functest/build_test.go

```diff
@@ -134,10 +134,9 @@ func TestBuildLabels(t *testing.T) {
 	}
 
 	// create a temporary dir to create the project and run the test
-	projectDir, err := ioutil.TempDir("", "appsody-build-labels-test")
-	if err != nil {
-		t.Fatal(err)
-	}
+	projectDir := cmdtest.GetTempProjectDir(t)
+	defer os.RemoveAll(projectDir)
+	t.Log("Created project dir: " + projectDir)
```

Comment:
We are doing these operations just below.

Reply:
removed the duplicates